src/ui/server/controllers/insurance/your-buyer/index.test.ts

```diff
@@ -27,9 +33,32 @@ describe('controllers/insurance/your-buyer/your-buyer-details', () => {
     jest.resetAllMocks();
   });
 
+  describe('PAGE_VARIABLES', () => {
+    it('should have correct properties', () => {
+      const expected = {
+        FIELDS: {
+          NAME: {
+            ID: NAME,
+            ...FIELDS.COMPANY_OR_ORGANISATION[NAME],
+          },
+          ADDRESS: {
+            ID: ADDRESS,
+            ...FIELDS.COMPANY_OR_ORGANISATION[ADDRESS],
+          },
+          COUNTRY: {
+            ID: COUNTRY,
+            ...FIELDS.COMPANY_OR_ORGANISATION[COUNTRY],
+          },
+        },
+      };
+
+      expect(PAGE_VARIABLES).toEqual(expected);
+    });
+  });
+
   describe('TEMPLATE', () => {
     it('should have the correct template defined', () => {
-      expect(TEMPLATE).toEqual(TEMPLATE);
+      expect(TEMPLATE).toEqual(TEMPLATES.INSURANCE.YOUR_BUYER.COMPANY_OR_ORGANISATION);
```

Comment:
do we need this test?

Reply:
Yes, to make sure the template variable has the correct template. This is then used in the get and post controllers